CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/editors/DateEditorFactory.java

```diff
@@ -69,9 +69,13 @@ protected LocalDate getValueFromControls() throws ControlsInvalidException {
             if (noValueCheckBox.isSelected()) {
                 return null;
             }
+            final String dateString = datePicker.getEditor().getText();
+            //The converter is being used here to try and determine if the entered date is a LocalDate
+            //It will throw an exception and won't convert it if its invalid
             try {
-                final String dateString = datePicker.getEditor().getText();
-                datePicker.getConverter().fromString(dateString);
+                if (!dateString.equals("")) {
```

Comment:
I would suggest using `String.isBlank` or even `StringUtil.isBlank` to additionally ignore null.

Reply:
@formalhaut69 I'd second what @cygnus-x-1 says with `StringUtil.isBlank`.

Side tracking a little and this is purely me sharing a tip I've learnt (from SQ) is in general if you have to compare a string with an object, putting the string to the left of the operation helps avoid an NPE - e.g. `"Some Text".equals(someObject)` 😄